src/Task/Git/CommitMessage.php

```diff
@@ -173,7 +173,7 @@ private function enforceTextWidth(ContextInterface $context)
         }
 
         foreach (array_slice($lines, 2) as $index => $line) {
-            if (isset($line[0]) && $line[0] == '#') {
+            if ($line !== '' && $line[0] === '#') {
```

Comment:
IMO the first check isn't needed because the second one decides to continue or not to continue with foreach

Reply:
If the line is empty, [0] does not exist and will throw a warning. It would function the same, the first check was to avoid a warning